src/edu/csus/ecs/pc2/core/model/Problem.java

```diff
@@ -211,8 +214,6 @@
     private VivaInputValidatorSettings vivaSettings = null;
 
 
-    
```

Comment:
Unable to leave a comment near:   public SandboxType getSandboxType() {
For backwards compatibility and to avoid NPEs should change to:
    public SandboxType getSandboxType() {
        if (sandboxType == null) {
            sandboxType  = SandboxType.NONE;
        }
        return sandboxType;
    }

Reply:
Agreed; fixed.